lib/inflector_extensions.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class String
+  def pluralize(count = nil, locale = :en)
+    locale = count if count.is_a?(Symbol)
+    if count == 1
+      dup
+    else
+      ActiveSupport::Inflector.pluralize(self, count, locale)
+    end
+  end
+end
+
+module InflectorExtensions
+  def pluralize(word, count = nil, locale = :en)
+    apply_inflections(word, inflections(locale).plurals, locale, count)
+  end

```

Comment:
add create specs also just for class `Inflections` and do something like `Inflections.pluralize('something', 2)   #=>  "somethings"`